File: app-02-layout.py
from dash import Dash, html
from dash import dcc # dash core component

import dash_bootstrap_components as dbc

app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])

# Initiatlize the app
app = Dash(__name__)

# Layout
app.layout = html.Div([
    html.H1('Hello Dash'),
    html.P('Dash converts Python classes into HTML'),
    html.P("This conversion happens behind the scenes by Dash's JavaScript front-end"),

    dcc.Markdown('''
    # Hello Dash

    Dash converts Python classes into HTML

    This conversion happens behind the scenes by Dash's JavaScript front-end
''')
])

# Server side callbacks/reactivity
# ...

# Run the app/dashboard
if __name__ == '__main__':
    app.run()

File: app-02-layout.R
# If you want the same port each time and enable hot reloading
# options(shiny.port = 8050, shiny.autoreload = TRUE)

library(shiny)
library(bslib)


# Layout
ui <- bslib::page_fluid(
  h1("Hello Shiny"),
  p("Shiny converts R classes into HTML"),
  p("This conversion happens behind the scenes by Dash's JavaScript front-end"),
)

# Server side callbacks/reactivity
server <- function(input, output, session) {}

# Run the app/dashboard
shinyApp(ui, server)
